File: src/test/java/com/jithesh/example/repository/TodoRepositoryTest.java
package com.jithesh.example.repository;

import org.assertj.core.api.Assertions;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
import org.springframework.test.context.junit4.SpringRunner;

import static org.assertj.core.api.Assertions.assertThat;

@DataJpaTest
@RunWith(SpringRunner.class)
public class TodoRepositoryTest {

  @Autowired
  private TodoRepository subject;

  @Test
  public void testRepositoryIsPopulated() {

    Assertions.assertThat(subject.findAll()).hasSize(3);
  }

}

File: src/test/kotlin/com/enablehr/challenge/kotlin/repository/TodoRepositoryTest.kt
package com.SampleApp.challenge.kotlin.repository

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
import org.springframework.test.context.junit4.SpringRunner

@DataJpaTest
@RunWith(SpringRunner::class)
internal class TodoRepositoryTest {

    @Autowired
    private lateinit var subject: TodoRepository

    @Test
    fun `test that repository exists`() {

        assertThat(subject.findAll()).hasSize(3)
    }
}
